jobfair-frontend/src/containers/forms/ResetPasswordForm/ResetPasswordForm.container.jsx

```diff
@@ -12,25 +12,35 @@ const ResetPasswordFormContainer = () => {
   const history = useHistory();
   const onFinish = async (values) => {
     const body = {
-      email: values?.email ? values.email : location?.state?.email,
+      email: location?.state?.email,
       otp: otpCode,
       newPassword: values.newPassword,
       confirmPassword: values.confirmPassword
     };
-    resetPasswordAPI(body)
-      .then(() => {
-        notification['success']({
-          message: `Your password has been change!.`,
-          duration: 1
-        });
-        history.push(PATH.LOGIN_PAGE);
-      })
-      .catch((err) => {
-        notification['error']({
-          message: `Error - ${err.response.data.message}`,
-          duration: 1
-        });
+    try {
+      await resetPasswordAPI(body);
+      Modal.success({
+        title: 'Reset password successfully !',
+        width: '30rem',
+        closable: true,
+        maskClosable: true,
+        onOk: () => history.push(PATH.LOGIN_PAGE),
+        keyboard: false,
+        content: (
+          <div style={{ display: 'flex', flexDirection: 'column', justifyContent: 'center' }}>
+            <Typography.Text>Your password has been updated</Typography.Text>
+          </div>
+        )
       });
+    } catch (e) {
+      Modal.error({
```

Comment:
is this modal error "props" used in many situations ? if so then create a default props for it (also for the success one)